pyt/cfg/stmt_visitor.py

```diff
@@ -9,7 +9,8 @@
     handle_aliases_in_init_files,
     handle_fdid_aliases,
     not_as_alias_handler,
-    retrieve_import_alias_mapping
+    retrieve_import_alias_mapping,
+    fully_qualify_alias_labels
```

Comment:
Nit: We tend to use alphabetical imports, e.g. `fully_qualify_alias_labels` would go after `as_alias_handler` here.